.gitignore

```diff
@@ -115,4 +115,14 @@ fabric.properties
 
 # Operator-sdk
 bin/*
+
```

Comment:
A PR adding tests shouldn't change the gitignore. I haven't seen any artifact while running `make test` but if they are generated and we don't want them, we should have a Jira issue and a PR for it.

Reply:
PTAL